### DISCUSSION TITLE
Rebranding

### DISCUSSION THREAD

**Discussion by bzglve** [bzglve/anyrun#3] [Ideas]
As the original repo seems to be dead, I'm thinking about some rebranding
Does it right from the point of view of ownership as original repo not archived or something
If there is nothing about it, what do you all think about `btrrun`/`grun`?

**Comment by stepanzak** [comment]
I like grun more than btrrun.

**Reply by bzglve** [reply]
What do you think about [UniRun](https://github.com/bzglve/unirun)?

**Reply by stepanzak** [reply]
That's even better!
